src/assets/scripts/rancher-desktop-guestagent.initd

```diff
@@ -31,5 +31,19 @@ start_pre() {
 EOF
 }
 
-set -o allexport
-if [ -f /etc/environment ]; then source /etc/environment; fi
+# shellcheck disable=SC2163
+if [ -f /etc/environment ]; then
+    while read -r line; do
+        # pam_env implementation:
+        # - '#' is treated the same as newline; terminates value
+        # - skip leading tabs and spaces
+        # - skip leading "export " prefix (only single space)
+        # - skip leading quote ('\'' or '"') on the value side
+        # - skip trailing quote only if leading quote has been skipped;
+        #   quotes don't need to match; trailing quote may be omitted
+        line="$(echo "$line" | sed -E "s/^[ \\t]*(export )?//; s/#.*//; s/(^[^=]+=)[\"'](.*[^\"'])?[\"']?$/\1\2/")"
+        [ -n "$line" ] && export "$line"
+    done </etc/environment
+fi
+
+true
```

Comment:
Please add a comment about this `true`. Does this still need to happen if `/etc/environment` doesn't exist?